contracts/Pair.sol

```diff
@@ -178,15 +178,14 @@ contract Pair is IPair {
 
     // Accrue fees on token0.
     function _update0(uint amount) internal {
-        if (hasGauge == false) {
+        if (!hasGauge) {
```

Comment:
Just a reminder that gauges can be killed. Not sure if we are dealing with this here or elsewhere but we should be.

Reply:
Kill gauge function should set has gauge to false.